backend/src/test/java/org/josh/backend/ConfigTest.java

```diff
@@ -24,13 +29,19 @@
 
 @WebMvcTest({Config.class})
 @AutoConfigureMockMvc
+@WithMockUser(username = "testUser", password = "secretPass3")
+@Import(JwtFilter.class)
 class ConfigTest {
 
     @Autowired
     private MockMvc mockMvc;
 
     private Config config;
 
+    @SuppressWarnings("unused")
+    @MockBean
+    private JwtService jwtService;
+
```

Comment:
Es funktioniert leider ohne nicht. 

```If you've added the @MockBean private JwtService jwtService; declaration to your test class and you're not using it directly in your test methods, you might see a warning that the variable is never used. This is expected behavior since you've created a mock bean for testing purposes and may not need to use it directly in your test methods.

However, creating the mock bean is important because it satisfies the dependency requirement for your JwtFilter class. The mock bean ensures that the required dependency is present in the Spring context during the tests, even if you're not interacting with it directly within the test methods.
```